docs_user/modules/proc_using-new-subnet-ranges.adoc

```diff
@@ -105,7 +101,10 @@ At this point, you can ping the adopted nodes from the {OpenShiftShort} nodes by
         subnetName: subnet1
 ----
 
```

Comment:
```suggestion
+
```
I think you need a plus sign at line 103 (between the code block and the note). The numbering of last two steps  starts over at 1, and I think it's because this plus sign is missing.